src/QsCompiler/Transformations/Monomorphization.cs

```diff
@@ -55,8 +195,6 @@ public override TypedExpression Transform(TypedExpression ex)
 
             if (types.Any() && ex.Expression is QsExpressionKind<TypedExpression, Identifier, ResolvedType>.CallLikeExpression call)
             {
-                string prefix = CreateTypePrefix(types);
-
                 // For now, only resolve identifiers of global callables
                 if (call.Item1.Expression is QsExpressionKind<TypedExpression, Identifier, ResolvedType>.Identifier id &&
```

Comment:
Seems like there should be a TODO here that I believe is addressed with the next PR?

Reply:
Yes the TODO is near the top of the function. It is addressed in #206 .